velox/benchmarks/basic/ComparisonConjunct.cpp

```diff
@@ -176,7 +176,7 @@ BENCHMARK(conjunctsNested) {
 
 int main(int argc, char* argv[]) {
   folly::Init init{&argc, &argv};
-  gflags::ParseCommandLineFlags(&argc, &argv, true);
+  ::gflags::ParseCommandLineFlags(&argc, &argv, true);
```

Comment:
I am still unsure why this needed the explicit namespace to disambiguate, maybe because of all the macro stuff in folly? I was unable to repro on [godbolt](https://godbolt.org/z/zhc7s8bfa)

Reply:
> If using folly::gflags it would complain that the namespace was not found 

So weird. Well, easy enough fix for now.